packages/workspace/src/command-line/nx-commands.ts

```diff
@@ -315,14 +315,25 @@ function withRunManyOptions(yargs: yargs.Argv): yargs.Argv {
       describe: 'Projects to run (comma delimited)',
       type: 'string',
     })
+    .option('tags', {
+      describe: 'Run projects having any of the given tags (comma delimited)',
+      type: 'string',
+    })
     .option('all', {
       describe: 'Run the target on all projects in the workspace',
       type: 'boolean',
       default: undefined,
     })
-    .check(({ all, projects }) => {
-      if ((all && projects) || (!all && !projects))
-        throw new Error('You must provide either --all or --projects');
+    .check(({ all, projects, tags }) => {
+      if (
+        (all && projects) ||
+        (projects && tags) ||
+        (all && tags) ||
+        (!all && !projects && !tags)
+      )
+        throw new Error(
+          'You must provide either --all or --projects or --tags'
+        );
```

Comment:
This can be simplified to:

```javascript
const suppliedArgs = !!all + !!projects + !!tags;
if (suppliedArgs === 1) {
  return true;
} else {
  // throw error
}
```

Optionally you may remove the redundant else here.

Reply:
✅ updated and rebased

Reply:
and fixed lint,  green on CI again